src/api/service/axios-jira-client.ts

```diff
@@ -0,0 +1,53 @@
+import {AxiosInstance} from 'axios'
+import {AxiosBitBucketClient} from './axios-bitbucket-client'
+
+type Issue = any
+
+export const FIELDS = Object.freeze({
+  ISSUE_TYPE: 'issuetype',
+})
+export class AxiosJiraClient {
+  readonly client: AxiosInstance
+
+  constructor(client: AxiosInstance) {
```

Comment:
constructor(readonly client: AxiosInstance) {}

Declaration above is not needed.

Reply:
TypeScript requires that block so it can properly validate/compile